### DISCUSSION TITLE
Programmatically select existing text and make it bold

### DISCUSSION THREAD

**Discussion by cozybuild** [ueberdosis/tiptap#1353] [General Questions] [answered]
Currently I'm looking for a solution to make a specific part of a text bold. I've got the editor with some content and a non-related JS event which should modify the editor content and make a specific range bold (for example from character 4 to character 7). Is there an easy way to do this?

**Comment by philippkuehn** [comment, marked as answer]
try this:
```js
editor
  .chain()
  .setTextSelection({
    from: 4,
    to: 7,
  })
  .setBold()
  .run()
```

**Reply by cozybuild** [reply]
And I struggled around with the editor state so much xD Thanks, works great & simple
